helm/msgsvc/charts/activemq/templates/deployment.yaml

```diff
@@ -65,6 +65,18 @@ spec:
       serviceAccountName: {{ include "activemq.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- if .Values.initContainers.enabled }}
```

Comment:
This is no longer required for 0.0.3 since the ActiveMQ container runs with root by default.

Reply:
removed.